client/src/language/languageClient.ts

```diff
@@ -66,7 +66,8 @@ export async function activateLanguageServer (context: ExtensionContext): Promis
       ]
     },
     initializationOptions: {
-      storagePath: context.storageUri?.fsPath
+      storagePath: context.storageUri?.fsPath,
+      extensionPath: context.extensionPath
```

Comment:
If using relative URIs to the client, this shouldn't be needed. It's up to you deciding what's the best approach.

Reply:
if we don't change the output destination where we compile the server to, then I can hardcode the path (from `bitbakeDocScanner.js` to `yocto-docs`). Is it close to what you mean?